Sources/SwiftSyntaxMacroExpansion/MacroReplacement.swift

```diff
@@ -287,12 +353,14 @@ extension MacroDeclSyntax {
   public func expand(
     _ node: some FreestandingMacroExpansionSyntax,
     definition: MacroExpansionExprSyntax,
-    replacements: [MacroDefinition.Replacement]
+    replacements: [MacroDefinition.Replacement],
+    genericReplacements: [MacroDefinition.GenericArgumentReplacement]
```

Comment:
We should default `genericReplacements` to `[]`. Otherwise this is a source-breaking change.

Also, I think we should have a release note entry for the API addition of `genericReplacements`.

Reply:
ok done

Reply:
Still open AFAICT

Reply:
Whoops, thank you

Reply:
Ok defaulted everywhere